package.json

```diff
@@ -255,37 +255,49 @@
       "default": "automatic",
       "order": 17
     },
+    "openerSplit": {
```

Comment:
On second though, since this is only support by pdf-view, perhaps something like `pdfViewSplitDirection` (or `pdfViewSplit`). Alternatively we simply add a description detailing that this option is currently only supported by the pdf-view opener? _The Principle of Least Surprise™_